src/ast/document-descriptor.ts

```diff
@@ -118,7 +118,7 @@ export class Document implements Feature {
     for (const scannedFeature of base.entities) {
       i += 1;
       if (scannedFeature instanceof ScannedImport) {
-        const scannedDoc = dependenciesByUrl.get(scannedFeature.url)!;
```

Comment:
Ah, just realized your comment above (https://github.com/Polymer/polymer-analyzer/commit/c3b133992aeccb7077a10c58fc206aa1fceb2645) answers this. TIL!